app/services/concerns/contracted.rb

```diff
@@ -36,8 +36,8 @@ module Contracted
     attr_accessor :contract_options
 
     def contract_class=(cls)
-      unless cls <= ::BaseContract
-        raise ArgumentError "#{cls.name} is not an instance of BaseContract."
+      unless cls <= ::ModelContract
```

Comment:
Why did you change this back? I'm pretty sure I originally moved this from ModelContract to BaseContract in case we have a service that doesn't depend on an AR model

Reply:
All I did was add a missing comma there after the `ArgumentError` as far as I'm concerned.

Reply:
But yeah apparently the class changed too there. Since I based this on release/11.2 originally this might've been mixed up. I'll change the class back.